client/tailscale/localclient.go

```diff
@@ -1057,6 +1057,29 @@ func (lc *LocalClient) NetworkLockDisable(ctx context.Context, secret []byte) er
 	return nil
 }
 
+// StreamFunnel returns an io.ReadCloser that streams serve/Funnel
+// connections made to the provided HostPort.
+//
+// If Funnel was not already enabled for the HostPort in the ServeConfig,
+// the backend enables it for the duration of the context's lifespan and
+// then turns it back off once the context is closed.
+// If Funnel was already enabled, the ServeConfig is untouched.
+func (lc *LocalClient) StreamFunnel(ctx context.Context, hp ipn.FunnelStreamRequest) (io.ReadCloser, error) {
```

Comment:
Maybe now that this is anchored to serve we rename this func? And the others with "Funnel" naming

And then I think some of the docs may be outdated